scripts/committee-oversight-crontasks

```diff
@@ -0,0 +1,5 @@
+# /etc/cron.d/committee-oversight-crontasks
+
+# Back up the hearings database at midnight GMT every Thursday.
```

Comment:
Could you update this comment to reflect the daily backup?

Reply:
Ah, great catch, thank you. I have fixed my careless mistake (and I'll be more mindful of my comments moving forward!)